apis/proto/v1/payload/payload.proto

```diff
@@ -418,6 +418,17 @@ message Object {
   message Locations {
     repeated Location locations = 1;
   }
+
+  message List {
+    message Request {}
+
+    message Response {
+      oneof payload {
+        Vector vector = 1;
+        google.rpc.Status status = 2;
+      }
+    }
+  }
```

Comment:
Please add some comments about the payload :)

Reply:
[add comment](https://github.com/vdaas/vald/pull/2145/commits/49814fd564c464e9620e9247adecc860a4df9bd5)